go.sum

```diff
@@ -353,10 +353,11 @@ go.etcd.io/bbolt v1.3.2/go.mod h1:IbVyRI1SCnLcuJnV2u8VeU0CEYM7e686BmAb1XKL+uU=
 go.etcd.io/etcd/api/v3 v3.5.0/go.mod h1:cbVKeC6lCfl7j/8jBhAK6aIYO9XOjdptoxU/nLQcPvs=
 go.etcd.io/etcd/client/pkg/v3 v3.5.0/go.mod h1:IJHfcCEKxYu1Os13ZdwCwIUTUVGYTSAM3YSwc9/Ac1g=
 go.etcd.io/etcd/client/v2 v2.305.0/go.mod h1:h9puh54ZTgAKtEbut2oe9P4L/oqKCVB6xsXlzd7alYQ=
-go.mongodb.org/atlas v0.8.1-0.20210618172518-81dd431f1e29 h1:NsxCCORBiWUFaE1p7V8AwQrIaQ3AqOoZ5EHbSLOoLqE=
-go.mongodb.org/atlas v0.8.1-0.20210618172518-81dd431f1e29/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
-go.mongodb.org/ops-manager v0.22.1-0.20210621094759-42feeae5f80d h1:UHyzBhNAMDl6klXkH5cVlHqpMeB+vkQTDjF4fhN+9RA=
-go.mongodb.org/ops-manager v0.22.1-0.20210621094759-42feeae5f80d/go.mod h1:dHSjoyergiwfMgS2HotlbmRRVx0x3bUfE0RTk4Bs//A=
+go.mongodb.org/atlas v0.8.1-0.20210628175115-05ebd0632b19/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
+go.mongodb.org/atlas v0.8.1-0.20210629072811-8369dddfaec6 h1:/QXjEFmPyLxyVFgb8LWroAF3jJfHR9ShnPgdiKo7SeI=
+go.mongodb.org/atlas v0.8.1-0.20210629072811-8369dddfaec6/go.mod h1:MMWDsc2akjTDSG4tVQrxv/82p3QbBnqeELbtTl45sbg=
```

Comment:
seems like an extra atlas here? can you please make sure the ops manager client has the same atlas client you are trying to use here and having run `go mod tidy`>